src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -0,0 +1,94 @@
+require "yast"
+
+require "cwm/table"
+
+require "y2partitioner/icons"
+
+module Y2Partitioner
+  module Widgets
+    # Table widget to represent given list of Y2Storage::BlkDevice.
+    class BlkDevicesTable < CWM::Table
+      include Yast::I18n
+
+      # @param blk_devices [Array<Y2Storage::BlkDevice>] devices to display
+      def initialize(blk_devices)
+        textdomain "storage"
+        @blk_devices = blk_devices
+      end
+
+      # headers of table
+      def header
+        [
+          # TRANSLATORS: table header, Device is physical name of block device
+          # like partition or disk e.g. "/dev/sda1"
+          _("Device"),
+          # TRANSLATORS: table header, size of block device e.g. "8.00 GiB"
+          Right(_("Size")),
+          # TRANSLATORS: table header, "F" stands for Format flag. Keep it short,
+          # ideally single letter. Keep in sync with F used later for format marker.
+          Center(_("F")),
```

Comment:
To ensure it is always in sync you can use this:

```ruby
# N_() just marks the text for the POT file,
# the value of the constant is not translated
FORMAT_FLAG = N_("F")
...
# _() runs the real translation
Center(_(FORMAT_FLAG))
...
device.exists_in_probed? ? "" : FORMAT_FLAG
```

But why the table header contains the translated "F" and the values in the column use untranslated "F"? Shouldn't be all translated?

BTW then you can use `FORMAT_FLAG` even in the help text to ensure it always match.